react/tic-tac-toe/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-mixed-operators */
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { func } from 'prop-types';
+
+import CustomInput from '../CustomInput';
+
+import { defaultEmailValidation, defaultPasswordValidation } from './utils';
+import styles from './styles.module.scss';
```

Comment:
Same. I'm pretty sure you don't need to add the `.module` extension for this to work.